tests/cli/env_test.go

```diff
@@ -113,3 +116,21 @@ func TestConfigConsistency(t *testing.T) {
 		},
 	})
 }
+
+func TestEnvConfig(t *testing.T) {
+	file, err := ioutil.TempFile(os.TempDir(), "acme-inc.env")
+	if err != nil {
+		t.Fatal(err)
+	}
+	content := []byte("FOO=bar\\nmoarbar\n")
+	file.Write(content)
+
+	run(t, []Command{
+		DeployCommand("latest", "v1"),
+		{
+			fmt.Sprintf("env-load %s -a acme-inc", file.Name()),
+			fmt.Sprintf("Updated env vars from %s and restarted acme-inc.", file.Name()),
+		},
+	})
+	defer os.Remove(file.Name())
```

Comment:
Pulled this change in as https://github.com/remind101/empire/commit/b55d8b4c4cfcfb68656c347c9891293990cecc7d.